src/SilverStripe/LocalGit/GitProcess.php

```diff
@@ -22,14 +20,29 @@ class GitProcess extends Process {
 	protected $identityFile;
 
 	/**
+	 * @var string Path to UserKnownHostsFile
+	 */
+	protected $knownHostsFile;
+
+	/**
 	 * Retrieves an absolute path to git.sh based on self::GIT_SSH.
 	 * The input path is assumed to be relative to the base dir of the rainforest project.
 	 *
 	 * @return string Path to git.sh
 	 */
 	static public function get_git_sh_path() {
-		// "realpath" dereferences symlinks.
-		$gitSh = dirname(dirname(dirname(realpath(__DIR__)))) . DIRECTORY_SEPARATOR . 'git.sh';
+		// git.sh must be composered into vendor/bin because whatever is passed to GIT_SSH needs the execute bit.
+		$gitSh = dirname(dirname(dirname(dirname(dirname(realpath(__DIR__))))))
```

Comment:
lol. Uh.. we could "undeep" the path to the source code in from ./src/SilverStripe/LocalGit/ to ./src and change https://github.com/silverstripeltd/localgit/blob/master/composer.json#L12 to read
```
"SilverStripe\\LocalGit\\": "src"
```

Reply:
Done! Looks better.